tensorflow/python/autograph/operators/control_flow.py

```diff
@@ -1344,7 +1344,7 @@ def if_stmt(cond, body, orelse, get_state, set_state, symbol_names, nouts):
       state that branches may mutate through side effects. This function is not
       needed and should not be called when dispatching to code matching Python's
       default semantics. This is useful for checkpointing to avoid unintended
-      side-effects when staging requires evaluating all code-paths.
+      side effects when staging requires evaluating all code-paths.
```

Comment:
`-` here is good (either form works so lt's not change needlessly)

Reply:
Thanks mihai for the review. updated as instructed.